File: app/src/main/java/com/example/mvvm_artichecture_sample/data/remote/apimodel/CountryModel.java
package com.example.mvvm_artichecture_sample.data.remote.apimodel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CountryModel {

    @SerializedName("code")
    @Expose
    private String code;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("states")
    @Expose
    private Object states;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Object getStates() {
        return states;
    }

    public void setStates(Object states) {
        this.states = states;
    }
}

File: app/src/main/java/com/example/mvvm_artichecture_sample/data/remote/apimodel/CountryModel.kt
package com.example.mvvm_artichecture_sample.data.remote.apimodel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class CountryModel {

    @SerializedName("code")
    @Expose
    var code: String? = null
    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("states")
    @Expose
    var states: Any? = null
}
